src/test/components/Home.test.js

```diff
@@ -127,7 +130,14 @@ describe('app/Home', function() {
     });
     await findByTestId('home-enable-popup-instructions');
 
-    fireFullClick(getByText('Enable Profiler Menu Button'));
+    fireFullClick(
+      getByText('Enable Firefox Profiler Menu Button', {
```

Comment:
Also I realized we changed the text itself (from `Profiler` to `Firefox Profiler`). Do you know if that's expected @CipherGirl ?

Reply:
I see that we use the full brand name. I think that's OK for now, we can move back to something shorter if we see it's a problem in the future.

Reply:
Initially, I changed that by mistake, that's why I asked if we should create another variable for `Profiler` only. But by `Profiler` we don't represent the app itself, right? It's better to use `Firefox Profiler` on Home Page I believe. Or the case here is different?

Reply:
Yeah, I think this is good :-)